README.md

```diff
@@ -1,8 +1,8 @@
-## Push Notification Plugin for Xamarin iOS and Android
+## Push Notification Plugin for Xamarin iOS and Android (Updated original package to use secure storage)
 Simple cross platform plugin for handling push notifications. Uses FCM for Android and APS for iOS.
 
 ### Setup
-* Available on NuGet: http://www.nuget.org/packages/Plugin.PushNotification [![NuGet](https://img.shields.io/nuget/v/Plugin.PushNotification.svg?label=NuGet)](https://www.nuget.org/packages/Plugin.PushNotification/)
+* Available on NuGet: https://www.nuget.org/packages/Plugin.PushNotification-updated/1.3.1 
```

Comment:
Here should be https://www.nuget.org/packages/Plugin.PushNotification

Reply:
I'll adjust that.